bogue.opam

```diff
@@ -50,3 +50,4 @@ build: [
   ]
 ]
 dev-repo: "git+https://github.com/sanette/bogue.git"
+available: [ (os-distribution != "opensuse-leap" | os-version >= 16) ]
```

Comment:
this is good for avoiding CI errors, but what if an OpenSuse user decides to manually install a recent version of SDL2?